x-pack/docs/en/security/auditing/output-logfile.asciidoc

```diff
@@ -2,87 +2,39 @@
-analytics it is strongly recommended to keep only the newer format.
-
-To turn off the deprecated output format, you can disable the logger in the
-`log4j2.properties` file:
-
-[source, properties]
---------------------------------------------------
-# change info to off
-# logger.xpack_security_audit_deprecated_logfile.level = info
-logger.xpack_security_audit_deprecated_logfile.level = off
---------------------------------------------------
-
-Alternatively, use the
-<<cluster-update-settings,cluster update settings API>> to dynamically
-configure the logger:
-
-[source,console]
---------------------------------------------------
-PUT /_cluster/settings
-{
-  "persistent": {
-    "logger.org.elasticsearch.xpack.security.audit.logfile.DeprecatedLoggingAuditTrail": "off"
-  }
-}
---------------------------------------------------
+The `logfile` audit output is the only output for auditing. It writes data to
+the `<clustername>_audit.json` file in the logs directory.
 
 NOTE: If you overwrite the `log4j2.properties` and do not specify appenders for
 any of the audit trails, audit events are forwarded to the root appender, which
 by default points to the `elasticsearch.log` file.
 
-
 [discrete]
 [[audit-log-entry-format]]
 === Log entry format
 
-The log entries in the `<clustername>_audit.json` file have the following format:
-
-- Each log entry is a one line JSON document and each one is printed on a separate line.
-- The fields of a log entry are ordered. However, if a field does not have a value it
-  will not be printed. The precise line pattern, together with the complete field
-  order, are specified in the `log4j2.properties` config file.
-- The log entry does not contain nested inner JSON objects, i.e. the doc is flat.
-- The field names follow a dotted notation to flatten inner objects.
-- A field's value can be a string, a number or an array of strings.
-- A field's value, a request body as well, will be escaped as per the JSON RFC 4627.
+The audit events are formatted as JSON documents, and each event is printed on a separate
+line in the `<clustername>_audit.json` file (the entries themselves do not contain the
+end-of-line delimiter).
+The audit event JSON format is somewhat particular, as *most* fields follow a dotted
+name syntax, are ordered, and contain non-null string values. This format creates a
+structured columnar aspect, similar to a CSV, that can be more easily inspected visually
+(compared to an equivalent nested JSON document).
+
+There are however a few attributes that are exceptions to the above format. The `put`,
+`delete`, `change`, `create` and `invalidate` attributes, which are only present for
+events with the `event.type: "security_config_change"` attribute, contain the *nested JSON*
+representation of the security change taking effect. The contents of the security config change
+are hence not splayed as top-level dot-named fields in the audit event document. That's because
```

Comment:
Is this a typo?
```suggestion
are hence not displayed as top-level dot-named fields in the audit event document. That's because
```

Reply:
Not a typo, but I obviously got entangled in the explanation. "Display" seems better, I'll use it.